.eslintignore

```diff
@@ -5,3 +5,4 @@ node_modules
 .env*
 *.config*
 .eslintrc.js
+cypress/
```

Comment:
ESLint kept complaining whenever I wanted to commit my Cypress test work, so I've shut ESLint up, by putting the whole `cypress/` folder in the `.eslintignore` file. This is probably a bad/stupid thing to do???

Reply:
And it turns out I'm breaking the whole build/linting run. 😄

Reply:
We should lint cypress tests files with ESLint indeed, there is even an [ESLint plugin for Cypress](https://github.com/cypress-io/eslint-plugin-cypress) 😅

Reply:
I tried setting up that ESLint for Cypress plugin locally, and that seems to work OK-ish, but not sure of the correct configuration.

I think it's probably best if you install and setup the Cypress ESLint plugin and the Cypress Testing Libary correctly yourself, and then push a new commit(s) with the new packages and configuration? I think otherwise I'll keep making unnecessary noise here. 😉 

I'm sorry for being a bit of an idiot with this whole "configuring Cypress and ESLint and Cypress Testing Libary) thing. 🙈 😄